templates/image-builder.yml

```diff
@@ -196,6 +196,7 @@ objects:
           c.request->'customizations'->'filesystem' as filesystem,
           c.request->'customizations'->'payload_repositories' as payload_repositories,
           c.request->'customizations'->'openscap' as openscap,
+          c.request->'customizations'->'files' as files,
```

Comment:
You might also want `c.request->'customizations'->'services' as services`, the first boot service will also show up in there.

Reply:
Yeah, that's what I was wondering about - is it more efficient to just rely on the `services` bit.
I guess we can try to dump both for now. I'll update the PR.